packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js

```diff
@@ -0,0 +1,62 @@
+import { CONTAINER_STATUS_OPTIONS } from './containers.constants';
+
+export default class PciStoragesContainersUsersController {
+  /* @ngInject */
+  constructor(
+    $translate,
+    atInternet,
+    CucCloudMessage,
+    PciStoragesColdArchiveService,
+  ) {
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciStoragesColdArchiveService = PciStoragesColdArchiveService;
+  }
+
+  $onInit() {
+    this.loadMessages();
+  }
+
+  loadMessages() {
+    this.messageHandler = this.CucCloudMessage.subscribe(
+      'pci.projects.project.storages.cold-archive.containers',
+      {
+        onMessage: () => this.refreshMessages(),
+      },
+    );
+  }
+
+  refreshMessages() {
+    this.messages = this.messageHandler.getMessages();
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getStatusActions(status) {
+    const { actions } = CONTAINER_STATUS_OPTIONS[status];
+    return actions;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getStatusClass(status) {
+    const { componentClass } = CONTAINER_STATUS_OPTIONS[status];
+    return componentClass;
+  }
+
+  getAction(action) {
+    const ACTIONS = {
+      add_user: this.goToAddUserContainer,
+      archive: this.archiveContainer,
+      restore: this.restoreContainer,
+      delete: this.deleteContainer,
```

Comment:
Actions here are not matching with actions in CONTAINER_STATUS_OPTIONS constant. Can you please check ?

Reply:
packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.constants.js

Reply:
This issue was fixed in an incoming PR. You're right I need to fix it in this PR.
Done.